_board/robohatmm1_m0.md

```diff
@@ -1,8 +1,8 @@
 ---
 layout: download
-board_id: "robohatmm1"
+board_id: "robohatmm1_m0"
```

Comment:
This will break downloads for the older versions. Would you rather keep the m0 name the same and just add the m4?

Reply:
It's ok to change.  Once PR #2014 gets merged in this won't break.  So this is on hold until I can fix up #2014 from adafruit/circuitpython

Reply:
PR 2014 has now been merged in.  So we should be good to go without breaking anything with this change.